system/src/ble_control_request_channel.cpp

```diff
@@ -346,9 +343,9 @@ class BleControlRequestChannel::HandshakeHandler {
     }
 
     int readPacket(const char** data, size_t* size) {
-        if (HAL_Timer_Get_Milli_Seconds() - timeStart_ >= HANDSHAKE_TIMEOUT) {
-            return SYSTEM_ERROR_TIMEOUT;
-        }
+//        if (HAL_Timer_Get_Milli_Seconds() - timeStart_ >= HANDSHAKE_TIMEOUT) {
```

Comment:
@sergeuz Are there any security implications?

Reply:
Since we no longer have user BLE services running while in listening mode, this should not be commented out.

Reply:
> Since we no longer have user BLE services running while in listening mode

We plan to allow user services to run while the device is in listening mode, so I guess it's fine to have this timeout disabled for now. Ideally, we should start tracking it when the setup service receives the first packet from the central device